### PR TITLE
[14.0][MIG] contract_operating_unit

### PR DESCRIPTION
**Pull request by kittiu**
Standard migration from #263

### CODE REVIEW

**Review by AaronHForgeFlow — COMMENTED** [review]
:+1:  Functional tests succeed.

Nevertheless, could you add a test that checks that the generated invoices belongs to the same OU?

File: contract_operating_unit/tests/test_contract_operating_unit.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                "company_id": company.id,
+                "company_ids": [(4, company.id)],
+                "operating_unit_ids": [(4, ou.id) for ou in operating_units],
+                "groups_id": [(6, 0, group_ids)],
+            }
+        )
+        return user
+
+    def test_contract_ou(self):
+        """Test Hr Contract Operating Unit"""
```
Comment (AaronHForgeFlow) [inline contract_operating_unit/tests/test_contract_operating_unit.py]: Sorry for this, I probably copied this from another module. Can you remove this comment?

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

File: contract_operating_unit/tests/test_contract_operating_unit.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply contract_operating_unit/tests/test_contract_operating_unit.py]: @kittiu  If you don't have time for this I can commit to your branch. Regards.

Comment (kittiu) [reply contract_operating_unit/tests/test_contract_operating_unit.py]: Please. I will merge in. Thanks.

Comment (AaronHForgeFlow) [reply contract_operating_unit/tests/test_contract_operating_unit.py]: Here it is https://github.com/ecosoft-odoo/operating-unit/pull/1 :)

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM!

**Review by AaronHForgeFlow — APPROVED** [review]
Functional test

**Pull request merged**